common/content/statusline.js

```diff
@@ -381,6 +399,23 @@ const StatusLine = Module("statusline", {
                     return [[name, fields[name].description] for (name of Object.keys(fields))];
                 },
             });
+
+        options.add(["statuslinevisibility"],
```

Comment:
Thanks for the hint. I didn't know how to add a short version and will do it now.